go/pkg/bertyprotocol/scenario_test.go

```diff
@@ -145,6 +145,9 @@ func TestScenario_MessageContactGroup(t *testing.T) {
 		// Add accounts as contacts
 		addAsContact(ctx, t, tps, tps)
 
+		// Waiting that secrets are sent
+		WaitingForContactReady(ctx, t, tps, tps)
```

Comment:
```suggestion
		waitForContactReady(ctx, t, tps, tps)
```

Reply:
Resolved because this code will be removed, see #2312 (comment)